docs/Problem-Solving-Examples-in-RxJava.md

```diff
@@ -61,7 +97,15 @@ def fibonacci = Observable.create({ observer ->
 But this is a little too much like ordinary linear programming. Is there some way we can instead create this sequence by composing together existing Observable operators?
 
 Here's an option that does this:
-````
+### Java
+```java
+Observable<Integer> fibonacci = Observable.fromArray(0)
+                                          .repeat()
```

Comment:
Please reformat this so that the operators do not trigger a horizontal scrollbar on smaller screens. For example, move `Observable.fromArray(0)` to an indented new line and line up the operators there.

```java
Observable<Integer> fibonacci = 
    Observable.fromArray(0)
              .repeat()
```

Reply:
done! Thank for reviewing 👍